### PR TITLE
V1 hardening

### PR DESCRIPTION
**Pull request by devcritter**
@rcrichton just some various edits and additions if you wouldn't mind taking a glance?

The only major thing are the addition of ATNA verifications for the e2e tests. After merging I think we're ready for the 1.0.0 release. What do you think?

Don't think there's much more to do: In terms of performance, the registry flow is network bound at this point, while the PnR flow is has 100ms or so hit on parsing the XDS XML, but I think that's something that we consider a major overhaul if we wanted to change that.

The last thing was Ryan Yates's CPU issue, but I can't replicate it. @BMartinos tried as well. Maybe you can just try on your system too? After that I think we'll have to deal with it as a separate issue.

### CODE REVIEW

File: src/test/java/org/openhim/mediator/denormalization/CSDRequestActorTest.java [hunk 1]
```diff
@@ -218,7 +218,7 @@ public void resolveFacilityIdentifier() throws Exception {
                 actor.tell(testMsg, getRef());
 
                 ResolveFacilityIdentifierResponse response = expectMsgClass(
-                        Duration.create(100, TimeUnit.MILLISECONDS),
+                        Duration.create(2000, TimeUnit.MILLISECONDS),
```
Comment (rcrichton) [inline src/test/java/org/openhim/mediator/denormalization/CSDRequestActorTest.java]: Are these 2 second timeouts? Maybe we should make these 60s. I think 60s is fair for most timeouts.

**Comment by rcrichton** [comment]
@devcritter looks great nice job! But one small comment about timeouts.

**Comment by devcritter** [comment]
@rcrichton sure, good idea. I upped the timeouts to 60s

**Comment by rcrichton** [comment]
Cool, looks good :+1:

**Pull request merged**